src/core/Chip/styles.css

```diff
@@ -0,0 +1,86 @@
+.container {
+    align-items: center;
+    background-color: #eceff1;
+    border-radius: 16px;
+    cursor: pointer;
+    display: inline-flex;
+    font-size: 14px;
+    height: 32px;
+    margin: 4px;
+    user-select: none;
+}
+
+.container:hover {
+    background-color: #e0f2f1;
+}
+
+.container.selected,
+.container.selected.static:hover {
+    background-color: #00796b;
+}
+
+.container.selected:hover {
+    background-color: #00695c;
+}
+
+.container.selected,
+.container.selected .icon,
+.container.container.selected .remove-icon {
+    color: #ffffff;
+}
+
+.container.static {
+    pointer-events: none;
+}
+
+.container.static:hover {
+    background-color: #eceff1;
+}
+
+.container.disabled {
+    cursor: not-allowed;
+    opacity: 0.3;
+}
+
+.container.disabled .remove-icon {
+    pointer-events: none;
+}
+
+.container.dragging {
+    box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2),
+        0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+
+.icon {
+    color: #494949;
+    font-size: 20px;
+    margin-left: 5px;
+}
+
+.image-icon {
+    border-radius: 50%;
+    height: 24px;
+    margin-left: 4px;
+    width: 24px;
+}
+
+.label {
+    margin: 0 12px;
+    max-width: 150px;
```

Comment:
- we should add a `prop` *overflow* with `default = true`
The Chips on the Dashboard are already > 150px. Then add the css `.container.overflow`

![screenshot 2018-10-18 at 16 35 05](https://user-images.githubusercontent.com/34527/47162189-ca6fa380-d2f3-11e8-8ca3-03a0b5a1ebef.png)

Reply:
4358c7d, opted for that `overflow: true` means _show long names_ and `overflow: false` to mean _truncate the long name_.